res/css/views/right_panel/_ThreadPanel.scss

```diff
@@ -201,7 +201,7 @@ limitations under the License.
         }
 
         &:hover .mx_EventTile_line {
-            box-shadow: unset !important; // don't show the verification left stroke in the thread list
+            box-shadow: inherit; // let the verification left stroke shown in the chat panel with a maximized widget
         }
```

Comment:
This file is ThreadPanel (i.e the list of threads) here the stroke shouldn't be shown

Reply:
The line was changed so that the left stroke would be displayed only in the chat panel, which is no longer the case.

The catch here is that `mx_ThreadPanel` class name is added to the chat panel with a maximized widget (`mx_TimelineCard`) as well.

![before](https://user-images.githubusercontent.com/3362943/167656248-d84814bc-04dd-4249-a2ae-fba8aa321fac.png)

This is a bug which should be fixed, whether incrementally or all at once.